forte/data/extractor/base_extractor.py

```diff
@@ -257,16 +257,25 @@ def extract(self, pack: DataPack,
         """
         pass
 
-    def remove_from_pack(self, pack: DataPack, instance: Annotation):
+    def pre_evaluation_action(self, pack: DataPack,
+                instance: Annotation):
         r"""Functionality: Remove the existing feature of the instance
         in a pack. We might remove an attribute under an entry or remove
         the entry itself directly, which will depend on the different
-        type of extractors.
+        type of extractors. This function is called before the evaluation
+        on a pack. After features are removed, new features predicted
+        from the model will be added to the pack.
 
         Overwrite instruction:
             1. Get all entries from one instance in the pack.
             2. Remove entries or remove some attributes of the entry. You
                 might use pack.delete_entry function or setattr to do this.
```

Comment:
Removing should not be the core instruction

Reply:
Got it.